src/server/common/test-helpers/snapshot-page.js

```diff
@@ -31,6 +32,10 @@ export const describePageSnapshot = ({
       for (const [key, value] of Object.entries(state)) {
         await session.setSectionState(key, value)
       }
+
```

Comment:
maybe check if it is possible to merge these two and keep passing just one argument as a full application state with a sections property in there?